packages/relay/tests/lib/sdkClient.spec.ts

```diff
@@ -2110,6 +2110,7 @@ describe('SdkClient', async function () {
     const fileCreateFee = Number(process.env.HOT_FIX_FILE_CREATE_FEE || 100000000); // 1 hbar
     const fileDeleteFee = Number(process.env.HOT_FIX_FILE_DELETE_FEE || 11000000); // 0.11 hbar
     const fileAppendFee = Number(process.env.HOT_FIX_FILE_APPEND_FEE || 120000000); // 1.2 hbar
+    const transactionRecordFee = 83333; // the precalcuated fee for transaction record queries
```

Comment:
Nit.  Maybe add the URL to the docs describing the fee calculation?

Reply:
Unfortunately, there isn't a URL to a document describing the calculation. It's based on a constant, `TX_RECORD_QUERY_COST_IN_CENTS = 0.01` cents, for which I’ve included a link to the fees calculator website.